data_display/views.py

```diff
@@ -81,6 +90,91 @@ def get_objects_by_table_and_sort(table_name, sort_by):
         'Not For Profits': (NotForProfits.objects.order_by(sort_by).values_list(), NotForProfits._meta.get_fields())
     }[table_name]
 
+def get_objects_by_table_and_filter(table_name, filter_table):
+    index = filter_table.find(',')
```

Comment:
Wait so you just take one input at a time? Is there anyway you can do some text parsing ahead of time so we can use more than one filter for things like AND queries or OR queries?

Eg. given the text input:

name=Ginny AND year=3

returns only students with both name Ginny and Year 3. 

To do the above you would need to do something like I was doing in string_display where you split the string according to commonly used delimiters (in this case we really only have '=', but you should prepare multiple types using regex). Then, for each element in the list that you split you should do what you're doing here.